modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/modals/PublishModal.js

```diff
@@ -40,6 +40,29 @@ export default function PublishModal({
 	const [displayDate, setDisplayDate] = useState(defaultDisplayDate);
 	const [dateError, setDateError] = useState('');
 	const [showErrorAlert, setShowErrorAlert] = useState(false);
+	const [isSubmitting, setIsSubmitting] = useState(false);
```

Comment:
@ambrinchaudhary, just a naming question. I don't see anywhere in js that we use is* or setIs* pattern

Reply:
You are totally right, I use the state because the idea was to disable the submit button (so there I would have used the `isSubmitting` variable),  but as this does not work, I will change it.

Reply:
Indeed we are using the [is**, setIs***] in portal, so I would keep it the same way... 
![Uploading Screenshot 2025-04-30 at 13.43.41.png…]()